metadata-protobuf/package.json

```diff
@@ -27,7 +27,8 @@
     "lint": "eslint ./src --ext .ts",
     "checks": "tsc --noEmit --pretty && prettier ./ --check && yarn lint",
     "format": "prettier ./ --write",
-    "prepublish": "yarn build"
+    "prepublishOnly": "npm run prepack",
+    "prepack": "npm run checks && npm run build"
```

Comment:
In general I'd advise running `checks` after `build`, because some of the checks can only be executed against a built code (ie. circular references check, or in case of `@joystream/types` for example - checks against autogenerated files). This is not really the case for `@joystream/metadata-protobuf` currently, but it would be good to have those executed in this order just in case some checks are added in the future or in case someone decides to use those scripts as a reference.

Reply:
resolved in 3eba5de